nautobot/dcim/utils.py

```diff
@@ -98,6 +99,7 @@ def get_all_network_driver_mappings():
         ("hier_config", HIERCONFIG_LIB_MAPPER_REVERSE),
         ("napalm", NAPALM_LIB_MAPPER_REVERSE),
         ("netmiko", NETMIKO_LIB_MAPPER_REVERSE),
+        ("netutils_parser", NETUTILSPARSER_LIB_MAPPER_REVERSE),
```

Comment:
Let's document this in `nautobot/docs/user-guide/core-data-model/dcim/platform.md` as a 2.0 addition to the `Platform` model functionality.